tests/opentelemetry/test_metrics.py

```diff
@@ -0,0 +1,92 @@
+import os
+
+from opentelemetry import version
+import pytest
+
+
+OTEL_VERSION = tuple(int(x) for x in version.__version__.split(".")[:3])
+
+
+def skipif(
+    exporter_installed: bool = False, exporter_not_installed: bool = False):
+    """
+    Returns a pytest skip marker based on OpenTelemetry version and exporter installation.
+    Parameters:
+    - exporter_installed: If True, skip tests that require OpenTelemetry exporters.
+    - exporter_not_installed: If True, skip tests that do not require OpenTelemetry exporters.
+    """
+    has_exporter = os.getenv("SDK_EXPORTER_INSTALLED", "").lower() in ("true", "1")
+    if exporter_installed and has_exporter:
+        return pytest.mark.skipif(True, reason="Tests not compatible with the opentelemetry exporters")
+    elif exporter_not_installed and not has_exporter:
+        return pytest.mark.skipif(True, reason="Tests only compatible with the opentelemetry exporters")
+    return pytest.mark.skipif(False, reason="No skip condition met for OpenTelemetry logs exporter tests")
+
+
+@skipif(exporter_installed=True)
+def test_otel_metrics_sdk_not_installed_by_default():
+    """
+    Test that the OpenTelemetry metrics exporter can be set up correctly.
+    """
+    from ddtrace.internal.opentelemetry.metrics import set_otel_meter_provider
+
+    # This should not raise an ImportError
+    set_otel_meter_provider()
+
+    # If the OpenTelemetry SDK is not installed
+    with pytest.raises(ImportError):
```

Comment:
Is the functionality being tested here implemented in ddtrace, or in python itself?

Reply:
The functionality of the metrics collection and export is done in the opentelemetry-python project and in the linked [system-tests PR](https://github.com/DataDog/system-tests/pull/4790) where we've set up the Python app appropriately to bring in the exporters and invoke each of the operations in the public API.

I know we can do more exhaustive testing in here, similar to what Munir is doing in his Logs API work where he's setting up a test server and validating the output requests. I'd like to omit that for now, if that's acceptable, since the OpenTelemetry Python SDK has quite a bit of testing. However, such testing will become critical if/when we decide to roll our own implementation.